packages/react/src/data-connect/useDataConnectMutation.test.tsx

```diff
@@ -1111,7 +1110,7 @@ describe("useDataConnectMutation", () => {
     await waitFor(() => {
       expect(result.current.isSuccess).toBe(true);
       expect(result.current.data).toHaveProperty("movie_insert");
-      expect(result.current.data?.movie_insert).toMatchObject({
+      expect(result.current.data?.ref.variables).toMatchObject({
```

Comment:
these are the only adjustments to the tests for when there is a factory function passed into the hook...the variables get to be available at ref.variables versus data.movie_insert